apps/discordmembers/discordmembers.star

```diff
@@ -0,0 +1,89 @@
+""")
+
+def main(config):
+    invite_id = config.get("invite_id", "r45MXG4kZc")
+
+    cache_key_members_count = "discord_members_%s" % invite_id
+    formatted_members_count = cache.get(cache_key_members_count)
+
+    cache_key_server_name = "discord_server_%s" % invite_id
+    server_name = cache.get(cache_key_server_name)
+
+    if formatted_members_count == None or server_name == None:
+        url = DISCORD_API_URL % invite_id
+        response = http.get(url)
+
+        if response.status_code != 200:
+            fail("Discord request failed with status %d", response.status_code)
+
+        body = response.json()
+
+        if body == None or len(body) == 0 or body["guild"] == None or len(body["guild"]) == 0:
+            formatted_members_count = "Not Found"
+            server_name = "Check your invite ID"
+        else:
+            formatted_members_count = "%s members" % humanize.comma(int(body["approximate_member_count"]))
+            server_name = body["guild"]["name"]
+            cache.set(cache_key_members_count, formatted_members_count, ttl_seconds = 240)
+            cache.set(cache_key_server_name, server_name, ttl_seconds = 240)
+
+    return render.Root(
+        child = render.Box(
+            render.Column(
+                expanded = True,
+                main_align = "space_evenly",
+                cross_align = "center",
+                children = [
+                    render.Row(
+                        expanded = True,
+                        main_align = "space_evenly",
+                        cross_align = "center",
+                        children = [
+                            render.Image(TWITTER_ICON),
+                            render.WrappedText(formatted_members_count),
+                        ],
+                    ),
+                    render.Marquee(
+                        width = 64,
+                        align = "center",
+                        offset_start = 10,
+                        child = render.Text(
+                            color = "#3c3c3c",
+                            content = server_name,
+                        ),
+                    ),
+                ],
+            ),
+        ),
+    )
+
+def get_schema():
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Text(
+                id = "invite_id",
+                name = "Invite ID",
+                icon = "userPlus",
+                desc = "Valid Discord Server Invite ID (Important: Set expiration to infinite)",
```

Comment:
It's quite important, actually. Otherwise, it stops working after the specified duration of the link 🤔 It might be self-explanatory, that if the ID stops working, the fetching stops, too. But still might be better to add that little explainer somewhere. (I can't do it as it's already merged, right? Can you?)